conans/client/cache/cache.py

```diff
@@ -188,6 +188,10 @@ def settings_path(self):
     def custom_commands_path(self):
         return os.path.join(self.cache_folder, EXTENSIONS_FOLDER, CUSTOM_COMMANDS_FOLDER)
 
+    @property
+    def custom_generators_path(self):
+        return os.path.join(self.cache_folder, EXTENSIONS_FOLDER, "generators")
```

Comment:
I am afraid that it would be better not to change the ``deploy`` folders, as we are aware of some users already using them, and it would be breaking, probably not worth it.

Regarding ``GENERATORS_EXTENSION_FOLDER`` I am tempted to go in the other direction, if no one else in the whole codebase uses the constant, then, tit is better encapsulated in the ``custom_generators_path``, so I was tempted to remove the ``CUSTOM_COMMANDS_FOLDER`` global const, and make it an implementation detail over ``custom_commands_path``, which is the abstraction to use, not the constant. I will double check that.

Reply:
Yeah, most of those consts, defined as public in the global module scope are unused in the rest of the codebase. I typically prefer to provide the less possible scope and indirection when it is artificial/unused/premature, so I'd probably propose an internal simplification PR for that (but I'll do it in another PR to keep this one minimal).

Reply:
Hi @memsharded,
Thanks for the additional insights. Regarding "deploy" vs. "deployers", would it be possible to look in both "deployers" and "deploy" but issues a deprecation warning for any files found in the older one? Thanks